src/providers/node/mod.rs

```diff
@@ -203,9 +219,24 @@ impl Provider for NodeProvider {
         // Start
         let start = NodeProvider::get_start_cmd(app, env)?.map(StartPhase::new);
 
-        let mut plan = BuildPlan::new(&vec![setup, install, build], start);
-        plan.add_variables(NodeProvider::get_node_environment_variables());
+        let mut phases = vec![setup, install, build];
+        if let Some(caddy) = SpaProvider::caddy_phase(app, env) {
+            phases.push(caddy); // insert after setup and before build
```

Comment:
Nit: remove the comment as it is not relevant anymore.

Does the build phase need to be updated to depend on the caddy? Or does it not matter the ordering?

Reply:
it doesn't matter